streaming/spark-streaming/examples/consumer/manifests/00_rbac_permissions.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: spark-operator
+  name: full-access-role
+rules:
+- apiGroups: [""]
+  resources: ["pods", "configmaps", "services", "endpoints", "persistentvolumeclaims", "events", "secrets", "serviceaccounts"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: ["apps"]
+  resources: ["deployments", "replicasets"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: ["batch"]
+  resources: ["jobs"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["roles", "rolebindings"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: full-access-rolebinding
+  namespace: spark-operator
+subjects:
+- kind: ServiceAccount
+  name: consumer-sa
+  namespace: spark-operator
+roleRef:
+  kind: Role
+  name: full-access-role
```

Comment:
See if you can use our existing spark-team-a team creation from spark-operator blueprint